client/src/+assets/user-publish-steps/main-page.tsx

```diff
@@ -176,6 +176,7 @@ export const UserPublishMultiStep: NextPage = () => {
     const onSubmitUserPublish = async() => {
         try {
 
+            setIsPublished(true)
             const findLocal = userPublish.asset_files.find(file => file.type === FileType.Local)
 
             if (findLocal != undefined){
```

Comment:
this can be like this
```typescript
if(!findLocal) {}
```
also always better to use `===` or `!==` instead of `!=` `==` because the first one compare also types

Reply:
done